package.json

```diff
@@ -171,6 +171,7 @@
     "sinon-chai": "^3.5.0",
     "standard-version": "^8.0.0",
     "strip-ansi": "^5.2.0",
+    "ts-node": "^8.10.1",
```

Comment:
added as devDependency for test case

Reply:
Let's not add it.

In previous comment I proposed to use fixture and when testing in fixture we may create a dummy `node_modules/ts-node.js` in a fixture folder, which should be a module that exposes a `register` method. Through that we may confirm it doesn't crash and that `register` is invoked